stetho/src/main/java/com/facebook/stetho/inspector/network/AsyncPrettyPrinterSchemaFileManager.java

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright (c) 2014-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+
+package com.facebook.stetho.inspector.network;
+
+import android.content.Context;
+import com.facebook.stetho.common.LogRedirector;
+import com.facebook.stetho.common.Util;
+
+import java.io.File;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.OutputStream;
+import java.net.URL;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.concurrent.atomic.AtomicInteger;
+
+public class AsyncPrettyPrinterSchemaFileManager {
```

Comment:
can this class be made `final` ? (in other words, there are no classes deriving from it, nor do we intend for that to happen?)